sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -29,7 +29,7 @@ public class BlobBaseClient
         /// <summary>
         /// The blob's primary <see cref="Uri"/> endpoint.
         /// </summary>
-        private readonly Uri _uri;
+        internal readonly Uri _uri;
```

Comment:
can't use virtual props in ctors. had to open this up internally. if better solution exists I'd appreciate hints.